chat-client/package.json

```diff
@@ -20,7 +20,7 @@
     "dependencies": {
         "@aws/chat-client-ui-types": "0.x.x",
         "@aws/language-server-runtimes-types": "0.x.x",
-        "@aws/mynah-ui": "^4.6.4"
+        "@aws/mynah-ui": "^4.9.x"
```

Comment:
Any specific reason to use both `^` and `x` here?

Reply:
No, I missed it when changing the version, will remove the `^` in a followup